README.md

```diff
@@ -157,19 +157,24 @@ rules:
 
 You may use the following shortcut or assemble your own config using the granular settings described below.
 
-Make sure you have installed [`@typescript-eslint/parser`] which is used in the following configuration. Unfortunately NPM does not allow to list optional peer dependencies.
+Make sure you have installed [`@typescript-eslint/parser`] and [`eslint-import-resolver-typescript`] which are used in the following configuration. Unfortunately NPM does not allow to list optional peer dependencies.
```

Comment:
(whoops, not sure how this comment got dropped, sorry)

```suggestion
Make sure you have installed [`@typescript-eslint/parser`] and [`eslint-import-resolver-typescript`] which are used in the following configuration.
```
let's just take this out - it's no longer true, npm should be lowercased, and we wouldn't want to have optional peer deps anyways.